iroh-util/src/lib.rs

```diff
@@ -229,6 +231,22 @@ pub fn increase_fd_limit() -> std::io::Result<u64> {
     Ok(soft)
 }
 
+/// Alias for a `Result` with the error type set to `UtilError`
+pub type Result<T> = result::Result<T, UtilError>;
+
+#[derive(Error, Debug)]
+pub enum UtilError {
```

Comment:
the convention is to call it just Error, so that in imports it will be iroh_utils::Error